src/driver/main.cpp

```diff
@@ -502,6 +503,7 @@ struct compiler
         ap(to_fp16, {"--fp16"}, ap.help("Quantize for fp16"), ap.set_value(true));
         ap(to_int8, {"--int8"}, ap.help("Quantize for int8"), ap.set_value(true));
         ap(to_fp8, {"--fp8"}, ap.help("Quantize for fp8e4m3fnuz type"), ap.set_value(true));
+        ap(to_int4, {"--int4"}, ap.help("Quantize for int4"), ap.set_value(true));
```

Comment:
Better to name it as `--int4-weights`

Reply:
Would you consider dot operations to contain 'weights'?

Reply:
Yes